res/skins/QMLDemo/DeckRow.qml

```diff
@@ -7,15 +7,68 @@ Item {
     property string leftDeckGroup // required
     property string rightDeckGroup // required
     property alias mixer: mixer
+    property bool minimized: false
 
     implicitHeight: mixer.height
+    states: [
+        State {
+            when: root.minimized
+            name: "minimized"
+
+            PropertyChanges {
+                target: mixer
+                visible: false
+            }
+
+            PropertyChanges {
+                target: root
+                implicitHeight: 56
+            }
+
+            AnchorChanges {
+                target: leftDeck
+                anchors.right: mixer.horizontalCenter
+            }
+
+            AnchorChanges {
+                target: rightDeck
+                anchors.left: mixer.horizontalCenter
+            }
+
+        },
+        State {
```

Comment:
is it possible to deduplicate this State by making the first one reversible?

Reply:
Added a comment why this is not possible.